Makefile

```diff
@@ -205,3 +196,72 @@ endif
 ifneq (,$(wildcard $(CURDIR)/package.json))
 	$(npm) run watch
 endif
+
+##
+## Tests
+##--------------------------------------
+
+.PHONY: test-syntax-php
+test-syntax-php:
```

Comment:
I don't think this is needed. Code style checks effectively do lint syntax checks for us.